kong/plugins/key-auth/handler.lua

```diff
@@ -154,14 +160,18 @@ local function do_authentication(conf)
   local cache = kong.cache
 
   local credential_cache_key = kong.db.keyauth_credentials:cache_key(key)
-  local credential, err = cache:get(credential_cache_key, nil, load_credential,
+  -- hit_level be 1 if stale value is propelled into L1 cache; so set a minimal `resurrect_ttl`
+  local credential, err, hit_level = cache:get(credential_cache_key, { resurrect_ttl = 0.001 }, load_credential,
                                     key)
+
+  kong.log.debug("credential hit_level: ", hit_level)
```

Comment:
Can the hit level be `nil` in which case I don't see point logging it.

Reply:
Ditto

Reply:
I just want to add a debug log to help debug. Can remove it as well.